integration/integration_test.py

```diff
@@ -525,6 +568,22 @@ def _get_buffers(self):
         ]
 
 
+class FixedSizeBinaryColumn(PrimitiveColumn):
+
+    def _encode_value(self, x):
+        return ''.join('{:02x}'.format(c).upper() for c in x)
+
+    def _get_buffers(self):
+        data = []
+        for i, v in enumerate(self.values):
+            data.append(self._encode_value(v if self.is_valid[i] else ""))
```

Comment:
This seems strange. Why are we using empty string to represent null value?

Reply:
We can probably just use any value for null instead of empty string. It should make the logic cleaner. (less if statements)

Reply:
oh... I put empty strings as null. sure, i can make do that.

Reply:
made the changes and simplified the logic inside JsonFileReader